packages/manager/src/features/Kubernetes/CreateCluster/ControlPlaneACLPane.tsx

```diff
@@ -31,11 +33,43 @@ export const ControlPlaneACLPane = (props: ControlPlaneACLProps) => {
     errorText,
     handleIPv4Change,
     handleIPv6Change,
-    ipV4Addr,
-    ipV6Addr,
     setControlPlaneACL,
   } = props;
 
+  const { clearErrors, control, setError, watch } = useFormContext();
+  const {
+    append: appendIPv4,
+    fields: ipv4Fields,
+    remove: removeIPv4,
+  } = useFieldArray({
+    control,
+    name: 'ipv4Addresses',
+  });
```

Comment:
Hmm. I was hoping we could do

```suggestion
  const {
    append: appendIPv4,
    fields: ipv4Fields,
    remove: removeIPv4,
  } = useFieldArray({
    control,
    name: 'control_plane.acl.addresses.ipv4',
  });
```

but it seems like react hook form isn't happy with it